mirrord/cli/src/config.rs

```diff
@@ -17,6 +18,8 @@ pub(super) enum Commands {
         path: String,
     },
     // Login(LoginArgs),
+    /// Operator commands eg. setup
+    Operator(Box<OperatorArgs>),
```

Comment:
https://docs.rs/clap/latest/clap/builder/struct.Command.html#method.hide
https://github.com/clap-rs/clap/issues/4353
Hide all operator stuff for now :)